new/INS1009Zammad.xml

```diff
@@ -0,0 +1,71 @@
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="INS1009Zammad" xml:lang="en" type="ins">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Žammadu Māryām</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MKr"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <t:langUsage xmlns:t="http://www.tei-c.org/ns/1.0">
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </t:langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="MKr" when="2024-02-28">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place sameAs="wd:Q124526652" type="monastery" subtype="institution">
+                    <placeName xml:lang="gez" xml:id="n1">ዠመዱ፡ ማርያም፡</placeName>
+                    <placeName xml:lang="gez" corresp="#n1">Žammadu Māryām</placeName>
+                    <placeName xml:lang="gez" xml:id="n2">ደብረ፡ ዘመዶ፡</placeName>
+                    <placeName xml:lang="gez" corresp="#n2">Dabra Zamaddo</placeName>
+                    <country ref="LOC3010Ethiop">Ethiopia</country>
+                    <region ref="LOC4259Lasta"/>
+                    <note>
+
+<ab type="description">According to tradition the monastery was founded by a local saint <persName ref="PRS2470Bartalom">Bartalomewos</persName> during the reign
+of <persName ref="PRS10303Yekunno">Yǝkunno ʾAmlāk</persName>.</ab>
+
+                    </note>
+
+
+                </place>
+            </listPlace><!---->
+            <div type="bibliography">
+                       <listBibl type="secondary">
+                           <bibl><ptr target="bm:Balicka2014Zammadu"/></bibl>
+                       </listBibl>
```

Comment:
https://betamasaheft.eu/Guidelines/?q=institution&id=placeOrInstitution - we need a relation to match to the place LOC6431Zammad